### PR TITLE
feat: add pireps/latest endpoint

### PR DESCRIPTION
**Pull request by Thien42**
@BossOfGames can you confirm if this would be the right approach to only fetch the latest pirep from a user?

I also want to make sure the response is an object and not an array.
I don't really have a way to test this at the moment, so thanks for your help!

### CODE REVIEW

**Comment by BossOfGames** [comment]
Yes, but there's a better way of doing it.
`$user->latest_pirep`

If you look at the \App\Model\User Model, you'll see a relationship setup for the latest pirep. I believe that code is what powers the latest pirep widget on the `/dashboard` for phpVMS.

**Comment by Thien42** [comment]
@BossOfGames thanks! Fixed the PR.

File: Http/Controllers/Api/PirepsController.php [hunk 1]
```diff
@@ -95,8 +95,7 @@ public function search(Request $request)
     public function latest(Request $request)
     {
         $user = User::find($request->get('pilotID'));
```
Comment (BossOfGames) [inline Http/Controllers/Api/PirepsController.php:97]: Is this meant to only be accessed by the current user? Or to grab any user's latest pirep? If it's only for the logged in user, do:

`$user = Auth::user();`

**Comment by BossOfGames** [comment]
I noticed that there wasn't a change made to the `/Http/Routes/api.php`

The route must be added here before it can be called by smartCARS, otherwise the call will fail outright with a 404 error.

File: Http/Controllers/Api/PirepsController.php [hunk 2]
```diff
@@ -95,8 +95,7 @@ public function search(Request $request)
     public function latest(Request $request)
```
Comment (BossOfGames) [inline Http/Controllers/Api/PirepsController.php:95]: Route must be added to `/Http/Routes/api.php` otherwise there will be a 404 error.

**Review by BossOfGames — CHANGES_REQUESTED** [review]

**Review by BossOfGames — APPROVED** [review]

**Pull request merged**